problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -181,5 +181,12 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+        
+		\prob Let $\vec u, \vec v \in \R^n$. Prove $|\vec u \cdot \vec v| \leq \|\vec u \|\|\vec v\|$, the \emph{Cauchy-Schwarz inequality}. Hint: use the geometric formula of the dot product of $\vec u, \vec v$.
```

Comment:
Use `\Norm{...}` and `\Abs{...}` . 

I think this problem needs a bit more scaffolding. Can you make it into a multi-part question?